ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -88,12 +88,11 @@ void RSkComponentImage::OnPaint(SkCanvas *canvas) {
     }
     /* TODO: Handle filter quality based of configuration. Setting Low Filter Quality as default for now*/
     paint.setFilterQuality(DEFAULT_IMAGE_FILTER_QUALITY);
-    setImageFilters(paint,imageProps,targetRect,frameRect,false,imageData->isOpaque());
+    filterForShadow(paint,imageProps,targetRect,frameRect,false,imageData->isOpaque());
```

Comment:
Comment given to change the argument name in setImageFilters, not to chnage the function name